cinemaAppFE/src/app/comment/comment.component.html

```diff
@@ -0,0 +1,14 @@
+
+
+<div id='container'>
+    
+    <div class='form'>
+        <h3>Leave the comment here</h3>
+         <input type="text" placeholder="Name" id="name"/><br/><br/>
```

Comment:
use ngModel to keep track of the data in the input insstead of jquery in the controller

         <input type="text" [(ngModel)]="formText" placeholder="Name" id="name"/><br/><br/>

formText is an property of the component class